src/main/java/org/opentripplanner/updater/vehicle_positions/VehiclePositionUpdaterRunnable.java

```diff
@@ -0,0 +1,59 @@
+package org.opentripplanner.updater.vehicle_positions;
+
+import com.google.common.base.Preconditions;
+import com.google.transit.realtime.GtfsRealtime.VehiclePosition;
+import org.opentripplanner.routing.edgetype.TripPattern;
```

Comment:
Remove redundant import

Reply:
Removed in bfa22c542c4831314e80c9e9b2f4d9fe2ee5ecc5